pkg/library/container/testdata/fixes-memory-limit-less-than-default-request.yaml

```diff
@@ -0,0 +1,24 @@
+name: "Gives useful error when memory limit is less than default request"
```

Comment:
Sorry for being late to the party, and that's fully OK you merged without my approval.

It seems to be copy/paste mistake. Is fixed in https://github.com/devfile/devworkspace-operator/pull/467

Reply:
Good catch, I should've waited :smile: 

https://github.com/devfile/devworkspace-operator/pull/469